bin/docker_build.sh

```diff
@@ -0,0 +1,15 @@
+# !/usr/bin/env bash
+
+ANYWAY_IMG_URL="${SERVER}/${DOCKER_REPOSITORY_ANYWAY}"
+NGINX_IMG_URL="${SERVER}/${DOCKER_REPOSITORY_NGINX}"
+
+echo ANYWAY_IMG_URL="${SERVER}/${DOCKER_REPOSITORY_ANYWAY}" 
+echo NGINX_IMG_URL="${SERVER}/${DOCKER_REPOSITORY_NGINX}" 
+
+docker pull "${ANYWAY_IMG_URL}:${SHA_TAG}" &&\
+if docker pull "${NGINX_IMG_URL}:${REF_TAG}"; then
+    CACHE_FROM=" --cache-from ${NGINX_IMG_URL}:${REF_TAG} "
+else
+    CACHE_FROM=""
+fi &&\
+docker build $CACHE_FROM -t "${NGINX_IMG_URL}:${SHA_TAG}" nginx_docker
```

Comment:
consider adding new line

Reply:
also, I'm not a docker expect, but why should we manually manage the construction from cache? and shouldn't we use `--build-arg BUILDKIT_INLINE_CACHE=1` if so?